lib/linguist/languages.yml

```diff
@@ -8388,6 +8388,13 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+templ:
+  type: programming
+  color: "#66D0DD"
+  extensions:
+  - ".templ"
+  ace_mode: text
+  language_id: 795579337
```

Comment:
You need to add `tm_scope` so the highlighter knows which grammar to use. The tests will fail without this too.

Reply:
Added, thanks. I did have this before but missed it with the final commit